proto/nexus/v1beta1/query.proto

```diff
@@ -38,3 +38,36 @@ message TransfersForChainResponse {
       [ (gogoproto.nullable) = false ];
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
+
+// FeeRequest represents a message that queries the transfer fees associated
+// to an asset on a chain
+message FeeRequest {
+  string chain = 1;
+  string asset = 2;
+}
+
+message FeeResponse { exported.v1beta1.FeeInfo fee_info = 1; }
+
+// TransferFeeRequest represents a message that queries the fees charged by
+// the network for a cross-chain transfer
+message TransferFeeRequest {
+  string source_chain = 1;
+  string destination_chain = 2;
+  bytes amount = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Uint",
+    (gogoproto.nullable) = false
+  ];
+  string asset = 4;
+}
+
+message TransferFeeResponse {
```

Comment:
I don't like this struct, it seems like it is doing way too much. Why can't it just return the fee?

Reply:
Fair enough, the caller will have to do more queries/computation to figure out the mins, but I understand the benefit of keeping this simple.